### DISCUSSION TITLE
Star rating range display not working as expected

### DISCUSSION THREAD

**Discussion by minasminas** [ppy/osu#16245] [Q&A] [answered]
In the multiplayer lounge it works correctly, but after joining the room it shows 0 stars.
This broke in osu!lazer version 2021.1218.0, before it worked correctly I think.

https://user-images.githubusercontent.com/81645139/147388239-c6c851bb-f1fd-45bf-967e-9bebbb9ab7df.mp4

Logs

[performance.log](https://github.com/ppy/osu/files/7775999/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7776000/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7776001/updater.log)
[network.log](https://github.com/ppy/osu/files/7776002/network.log)

**Comment by bdach** [comment, marked as answer]
this was regressed semi-intentionally as per https://github.com/ppy/osu/pull/15879#discussion_r759665038, but i'll open a tracking issue for it

**Reply by bdach** [reply]
#16251
